requirements.txt

```diff
@@ -6,6 +6,8 @@ networkx
 git+https://github.com/lsst/resources@main#egg=lsst-resources
 git+https://github.com/lsst/daf_butler@main#egg=lsst-daf-butler
 git+https://github.com/lsst/utils@main#egg=lsst-utils
-git+https://github.com/lsst/pipe_base@main#egg=lsst-pipe-base
+git+https://github.com/lsst/pipe_base@tickets/DM-41606#egg=lsst-pipe-base
 git+https://github.com/lsst/pex_config@main#egg=lsst-pex-config
 sqlalchemy
+types-psutil
+types-psycopg2
```

Comment:
FYI types go in a file called `types.txt` and not in `requirements.txt`. The mypy action looks in a special place because they aren't real requirements.

I'm a bit confused about the psutil one since psutil is only used by test code and so shouldn't get involved in mypy (it also is in the wrong place in pyproject.toml -- looks like it's been in the wrong place since the beginning so not related to this PR). Also, psycopg2 is not used anywhere in ctrl_mpexec so maybe I need

Reply:
AH, ok. I will move these edits to types.txt. I googled the mypy error and the first suggestion I tried was `mypy --install-types` and those were what it installed. They didn't end up resulting in a "pass" from mypy here, though. The other suggested line, `python3 -m pip install types-PyYAML`, did work, and that was what installed `types-PyYAML`. So, I'll do a hard reset and move that line to types.txt. Thank you for the help!!

Reply:
@timj it looks like that worked! Thanks for the linting help :)